### PR TITLE
German translations - WIP

### PR DESCRIPTION
**Pull request by analognightmare**
I worked on some German translations, but have officially lived in Canada long enough to find German quite clunky so I need some :eyes: on this from someone else. Therefore, this is a draft :slightly_smiling_face:

### CODE REVIEW

**Comment by phoet** [comment]
@analognightmare i would recommend throwing the text into https://mentor.duden.de/

File: src/locale/translations/de.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    "DailyShareDetailed": "Das Teilen deiner Daten hilft anderen herauszufinden ob sie mit COVID-19 in Kontakt kamen.",
+    "ShareRandomIDsCTA": "Teile deine zufälligen IDs",
+    "BluetoothDisabled": "Bluetooth ist aus",
+    "EnableBluetoothCTA": "COVID Shield benötigt Bluetooth um zufällige IDs zu teilen und sammeln.",
+    "TurnOnBluetooth": "Schalte Bluetooth an",
+    "ExposureNotificationsDisabled": "COVID-19 Kontakt Benachrichtungen sind aus",
+    "ExposureNotificationsDisabledDetailed": "COVID-19 Kontakt Benachrichtungen werden benötigt damit COVID Shield funktioniert.",
+    "EnableExposureNotificationsCTA": "Schalte COVID-19 Kontakt Benachrichtigungen ein",
+    "NoConnectivity": "Keine Internet Verbindung",
+    "NoConnectivityDetailed": "COVID Shield braucht Internet um mögliche Kontakte zu finden.",
```
Comment (MrMarvin) [inline src/locale/translations/de.json:26]:
```suggestion
    "NoConnectivityDetailed": "COVID Shield benötigt eine Internetverbindung um mögliche Kontakte zu finden.",
```

File: src/locale/translations/de.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    "DailyShare": "Teile deine neuen zufälligen IDs",
+    "DailyShareDetailed": "Das Teilen deiner Daten hilft anderen herauszufinden ob sie mit COVID-19 in Kontakt kamen.",
+    "ShareRandomIDsCTA": "Teile deine zufälligen IDs",
+    "BluetoothDisabled": "Bluetooth ist aus",
+    "EnableBluetoothCTA": "COVID Shield benötigt Bluetooth um zufällige IDs zu teilen und sammeln.",
+    "TurnOnBluetooth": "Schalte Bluetooth an",
+    "ExposureNotificationsDisabled": "COVID-19 Kontakt Benachrichtungen sind aus",
+    "ExposureNotificationsDisabledDetailed": "COVID-19 Kontakt Benachrichtungen werden benötigt damit COVID Shield funktioniert.",
+    "EnableExposureNotificationsCTA": "Schalte COVID-19 Kontakt Benachrichtigungen ein",
+    "NoConnectivity": "Keine Internet Verbindung",
```
Comment (MrMarvin) [inline src/locale/translations/de.json:25]:
```suggestion
    "NoConnectivity": "Keine Internetverbindung",
```

File: src/locale/translations/de.json [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    "SignalDataSharedDetailed": {
+      "One": "Du bekommst täglich eine Nachricht, die dich fragt deine zufälligen IDs zu teilen, für den nächsten {number} Tag",
+      "Other": "Du bekommst täglich eine Nachricht, die dich fragt deine zufälligen IDs zu teilen, für die nächsten {number} Tage"
+    },
+    "SignalDataSharedCTA": "Verfolge deine Symptome",
+    "SymptomTrackerUrl": "https://ca.thrive.health/covid19app/action/88a5e9a7-db9e-487b-bc87-ce35035f8e5c?from=/home&navigateTo=/tracker/complete",
+    "DailyShare": "Teile deine neuen zufälligen IDs",
+    "DailyShareDetailed": "Das Teilen deiner Daten hilft anderen herauszufinden ob sie mit COVID-19 in Kontakt kamen.",
+    "ShareRandomIDsCTA": "Teile deine zufälligen IDs",
+    "BluetoothDisabled": "Bluetooth ist aus",
```
Comment (MrMarvin) [inline src/locale/translations/de.json:19]:
```suggestion
    "BluetoothDisabled": "Bluetooth ist ausgeschaltet",
```

File: src/locale/translations/de.json [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    "EnableBluetoothCTA": "COVID Shield benötigt Bluetooth um zufällige IDs zu teilen und sammeln.",
+    "TurnOnBluetooth": "Schalte Bluetooth an",
```
Comment (MrMarvin) [inline src/locale/translations/de.json:21]:
```suggestion
    "TurnOnBluetooth": "Bluetooth anschalten",
```

File: src/locale/translations/de.json [hunk 5]
```diff
@@ -0,0 +1,158 @@
+{
+  "Home": {
+    "NoExposureDetected": "Kein Kontakt mit COVID-19 gefunden",
+    "NoExposureDetectedDetailed": "Den zufälligen IDs nach zu urteilen, warst du in den letzten 14 Tagen nicht in der Nähe einer Person, die positiv auf COVID-19 getestet wurde. ",
+    "ExposureDetected": "Du bist eventuell mit COVID-19 in Kontakt gekommen",
+    "ExposureDetectedDetailed": "Den zufälligen IDs nach zu urteilen, warst du in den letzten 14 Tagen in der Nähe einer Person, die positiv auf COVID-19 getestet wurde. ",
+    "SeeGuidance": "Lies Kanadas COVID-19 Handlungsempfehlung",
```
Comment (MrMarvin) [inline src/locale/translations/de.json:7]:
```suggestion
    "SeeGuidance": "Lies die kanadische COVID-19 Handlungsempfehlung",
```

File: src/locale/translations/de.json [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+    "GuidanceUrl": "https://www.canada.ca/en/public-health/services/publications/diseases-conditions/covid-19-how-to-isolate-at-home.html",
+    "SignalDataShared": "Danke, dass du hilst die Ausbreitung zu verlangsamen.",
+    "SignalDataSharedDetailed": {
+      "One": "Du bekommst täglich eine Nachricht, die dich fragt deine zufälligen IDs zu teilen, für den nächsten {number} Tag",
```
Comment (MrMarvin) [inline src/locale/translations/de.json:11]:
```suggestion
      "One": "Du bekommst eine Nachricht, die dich fragt deine zufälligen IDs zu teilen, im Laufe des nächsten Tages",
```

File: src/locale/translations/de.json [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+      "Other": "Zuletzt vor {number} Tagen aktualisiert"
+    }
+  },
+  "OverlayClosed": {
+    "SystemStatus": "COVID Shield ist ",
+    "SystemStatusOn": "AN",
+    "SystemStatusOff": "AUS",
+    "NotificationStatus": "Push Nachrichten sind ",
+    "NotificationStatusOff": "AUS",
+    "TapPrompt": "Klick für mehr Informationen"
```
Comment (MrMarvin) [inline src/locale/translations/de.json:47]: "Klick" vs. "Tap"? Ich glaube der "tappe hier" hat sich genug durchgesetzt um als deutsches wort durchzugehen.

File: src/locale/translations/de.json [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+    "EnterCodeCardBody": "Hat ein Gesundheitsexperte dich gefragt deinen COVID Shield code einzugeben?",
+    "EnterCodeCardAction": "Code eingeben",
+    "NotificationCardStatus": "Push Nachrichten sind ",
+    "NotificationCardStatusOff": "AUS",
+    "NotificationCardBody": "Du wirst keine Push Nachrichten bekommen, wenn du eventuell mit COVID-19 in Kontakt kamst. Zeitnahe Benachrichtung über potenziellen Kontakt is wichtig um die Ausbreitung von COVID-19 zu verlangsamen",
+    "NotificationCardAction": "Schalte Push Nachrichten",
+    "ExposureNotificationCardStatus": "COVID-19 Kontakt Benachrichtigungen sind ",
+    "ExposureNotificationCardStatusOff": "AUS",
+    "ExposureNotificationCardBody": "COVID-19 Kontakt Benachrichtigungen werden benötigt damit COVID Shield funktioniert.",
+    "ExposureNotificationCardAction": "Schalte COVID-19 Kontakt Benachrichtigungen ein"
```
Comment (MrMarvin) [inline src/locale/translations/de.json:64]:
```suggestion
    "ExposureNotificationCardAction": "COVID-19 Kontakt Benachrichtigungen einschalten"
```

File: src/locale/translations/de.json [hunk 9]
```diff
@@ -0,0 +1,158 @@
...
+    "Title": "Deine Daten sind privat",
+    "Body": "COVID Shield ist datensparsam.",
+    "Body2": "Deine zufälligen IDs werden nur auf deinem Handy gespeichert außer du entscheidest dich sie zu teilen.",
+    "Body3": "Das Benutzen von COVID Shield ist freiwillig. Du kannst deine zufälligen IDs jederzeit löschen."
+  },
+  "OnboardingStart": {
+    "Title": "Schütze dich und andere",
+    "Body1": "Werde benachrichtigt wenn du potenziell mit COVID-19 in Kontakt kamst.",
+    "Body2": "Wenn du COVID-19 positiv getestet wurdest, kannst du anonym deine Daten teilen, sodass andere über potenzielle Ansteckung informiert werden können.",
+    "TutorialAction": "Lerne wie COVID Shield funktioniert"
```
Comment (MrMarvin) [inline src/locale/translations/de.json:120]:
```suggestion
    "TutorialAction": "Erfahre wie COVID Shield funktioniert"
```

File: src/locale/translations/de.json [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+  },
+  "Privacy": {
+    "Title": "Datenschutzerklärung",
+    "Close": "Schließen"
+  },
+  "ThankYou": {
+    "Title": "Danke für deine Hilfe",
+    "Body": "COVID-19 Kontakt Benachrichtigungen sind an. Du wirst benachrichtigt, wenn COVID Shield einen möglichen Kontakt mit COVID-19 entdeckt.",
```
Comment (MrMarvin) [inline src/locale/translations/de.json:128]:
```suggestion
    "Body": "COVID-19 Kontakt Benachrichtigungen sind eingeschaltet. Du wirst benachrichtigt, wenn COVID Shield einen möglichen Kontakt mit COVID-19 entdeckt.",
```

File: src/locale/translations/de.json [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+    "Dismiss": "Ablehnen"
+  },
+  "DataUpload": {
+    "Cancel": "Abbrechen",
+    "FormIntro": "Bitte gib deinen 8 stelligen COVID Shield Code ein.",
+    "Action": "Code abschicken",
+    "InfoSmall": "Deine zufälligen IDs werden nicht geteilt, bis du im nächsten Schritt zustimmst.",
+    "ConsentTitle": "Teile deine zufälligen IDs",
+    "ConsentBody": "Ein Gesundheitsexperte hat dir erlaubt deine zufälligen IDs zu teilen.",
+    "ConsentBody2": "If you share your random IDs, others with COVID Shield who have been near you in the past 14 days will receive a notification that they have possibly been exposed. This notification will not include any information about you.",
```
Comment (MrMarvin) [inline src/locale/translations/de.json:138]:
```suggestion
    "ConsentBody2": "Wenn du zufällige IDs teilst, können andere die sich in der Vergangenheit in deiner Nähe aufgehalten haben eine Nachricht erhalten. Diese Nachricht ist anonym und enthält keine Information über dich.",
```

File: src/locale/translations/de.json [hunk 12]
```diff
@@ -0,0 +1,158 @@
...
+    "ConsentBody3": "You will be notified once per day when new random IDs are available to be shared. You must grant permission each time for the random IDs to be shared. Sharing is voluntary.",
```
Comment (MrMarvin) [inline src/locale/translations/de.json:139]:
```suggestion
    "ConsentBody3": "Du wirst täglich daran erinnert, deine neuen zufälligen IDs zu teilen. Du musst jedes mal zustimmen, um zu teilen. Teilen passiert freiwillig.",
```

**Comment by kpeatt** [comment]
image

image

**Comment by analognightmare** [comment]
@MrMarvin Vielen Dank! Zu lange nicht mehr in Deutschland gelebt und habe deshalb nicht mehr so ein gutes Gefuehl dafuer was geht und was nicht :)

**Comment by analognightmare** [comment]
Waiting for #25 for the language picker.

**Comment by emilybulger** [comment]
It's not clear to me what remains to be done to be able to close this issue. @analognightmare do you know? Anything I can do to help?

**Comment by analognightmare** [comment]
I will resolve the conflicts and then it should be ready to merge, will get around to that during the weekend :slightly_smiling_face:

**Pull request closed without merging**